### PR TITLE
Create aws-kafka.md

### PR DESCRIPTION
**Pull request by lohking**
docs for kafka>mixpanel

### CODE REVIEW

**Comment by ranic** [comment]
thanks for doing this @lohking! let's not merge this until we have someone from engineering tal at this first

**Comment by myronkaifung** [comment]
Awaiting review from engineering

**Review by myronkaifung — CHANGES_REQUESTED** [review]
I would recommend double-checking the file path again before merging as the nesting has changed since the release of new nav (suggestion: docs/pages/docs/tracking-methods/integrations/). Other than that, please have someone from Eng review the content itself prior to merging as per Vijay's suggestion

File: pages/docs/tracking/integrations/aws-kafka.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        if msg.error():
+            if msg.error().code() == KafkaError._PARTITION_EOF:
+                continue
+            else:
+                print(msg.error())
+                break
+
+        # Send to Mixpanel
+        event_data = msg.value().decode('utf-8')
+        mp.track('some_user_id', 'My Event', {'message': event_data})
```
Comment (csiden) [inline pages/docs/tracking/integrations/aws-kafka.md]: This looks like it's showing a whole item taken off kafka being set as a single event property called `message`? I think that would be a bad example to give, I know this is for hypothetical data, but maybe we could show an example of taking a few properties off of `event_data` and setting them as explicit properties on the event being tracked so people following the doc know not to track the entire event as a single property?

**Comment by myronkaifung** [comment]
Hi @lohking wanted to check in on this PR. Are we still looking for Eng review for this PR?

**Comment by lohking** [comment]
@myronkaifung sorry i missed this! Yes, currently awaiting Eng review to PR.

**Comment by myronkaifung** [comment]
> @myronkaifung sorry i missed this! Yes, currently awaiting Eng review to PR.

Awesome thanks. Let me know if we need to flag this for attention. Let's get this published!

**Comment by lohking** [comment]
We've gotten customers asking for docs kafka. Could I trouble you to escalate this to the right person to review? :)​

**Comment by myronkaifung** [comment]
@lohking added a comment about the event_data payload. Gonna deploy this today

**Review by myronkaifung — APPROVED** [review]
Added comment to code example. Let's ship for now and update the payload later on

**Pull request merged**